<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShiftEditForm</class>
 <widget class="QWidget" name="ShiftEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>424</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Time-Track</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ClockoutForm{

background-color:#E5EAE7;
}
*{
font-family:&quot;Calibri&quot;;
font-size: 14px;

}



QStackedWidget{
margin:0px;
padding:0px;
}


QWidget{

margin:0px;
padding:0px;
}
QWidget#bigwidger{
background-color:#E5EAE7;
border:none;
}



QTableWidget{
border:none;
}
QTextEdit{
border:none;
}


QPushButton{
padding:3px;
border:none;
background-color:#FFFFFF;
}
QPushButton:hover:!pressed{
background-color:#E4E9EB;
}


QDateTimeEdit{
border:none;
background-color:#FFFFFF;
}
QDateEditTime:hover{
border:none;
background-color:#E4E9EB;
}
QDateEditTime::drop-down{
border:none;
}


QComboBox{
border:none;
background-color:#FFFFFF;
}
QComboBox:hover {
background-color:#E4E9EB;
}
QComboBox:drop-down{
border:none;
}


QRadioButton{
padding:0px;
margin:0px;
}


QGroupBox {
    border: 0px solid gray;
	
    margin-top: 0.5em;
	margin-right:0px;
	margin-left:0px;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="bigwidger" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QWidget" name="left" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QDateTimeEdit" name="DateTime1">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="DateTime2">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Name">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Project:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Projects">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Sub-Project:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Items">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="editable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Time Worked:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="Times">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="displayFormat">
                <string>hh:mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="Add">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Delete">
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Lunch Taken:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTimeEdit" name="Lunch">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumTime">
                <time>
                 <hour>1</hour>
                 <minute>0</minute>
                 <second>0</second>
                </time>
               </property>
               <property name="currentSection">
                <enum>QDateTimeEdit::HourSection</enum>
               </property>
               <property name="displayFormat">
                <string>hh:mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="right" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableWidget" name="Sections">
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>30</number>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="DescriptionWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>35</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="text">
                 <string>Notes:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="Description">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>26</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="Description_Check">
                <property name="text">
                 <string/>
                </property>
                <property name="tristate">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="bottom" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Time Left:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeLeft">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="RefreshButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="FinishedButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="text">
            <string>Finish</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CancelButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>DateTime1</tabstop>
  <tabstop>DateTime2</tabstop>
  <tabstop>Name</tabstop>
  <tabstop>Projects</tabstop>
  <tabstop>Items</tabstop>
  <tabstop>Times</tabstop>
  <tabstop>Add</tabstop>
  <tabstop>Delete</tabstop>
  <tabstop>Lunch</tabstop>
  <tabstop>RefreshButton</tabstop>
  <tabstop>FinishedButton</tabstop>
  <tabstop>CancelButton</tabstop>
  <tabstop>Description</tabstop>
  <tabstop>Sections</tabstop>
  <tabstop>Description_Check</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
